recipes/amazon-braket-ocean-plugin/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "amazon-braket-ocean-plugin" %}
+{% set version = "1.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/amazon-braket-ocean-plugin-{{ version }}.tar.gz
+  sha256: dd6923d8b654775f3b896c8edf97295179ef68cc9ee19bfa47e746c456f4e6e8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7.2
```

```suggestion
    - python >=3.7
```
https://github.com/aws/amazon-braket-ocean-plugin-python/blob/cba5c10b6d65019801f68c49745ac362cf0991fe/setup.py#L26

Update in both host/run.